src/components/ContactCard.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="contact-card">
+    <h2 class="contact-card__heading">Contact us</h2>
+    <div class="contact-card__main-wrapper">
+      <div class="contact-card__wrapper">
+        <IconLocation class="contact-card__icon" />
+        <p class="contact-card__location">
+          Monterail Sp. z o.o. <br />
+          ul. Oławska 27-29 <br />
+          50-123 Wrocław <br />
+          Poland
+        </p>
+      </div>
+      <div class="contact-card__wrapper">
+        <IconEmail class="contact-card__icon" />
+        <p class="contact-card__email">hello@monterail.com</p>
+      </div>
+      <div class="contact-card__wrapper">
+        <IconPhone class="contact-card__icon" />
+        <p class="contact-card__phone">+48 533 600 136</p>
```

Comment:
it may be a good idea to make email and phone number links using relevant protocols https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-href